docs/source/transducer.rst

```diff
@@ -8,16 +8,11 @@ torchaudio.prototype.transducer
 
 .. note::
 
-    The RNN transducer loss is a prototype feature, see `here <https://pytorch.org/audio>`_ to learn more about the nomenclature. It is only available within the nightlies, and also needs to be imported explicitly using: :code:`from torchaudio.prototype.transducer import rnnt_loss, RNNTLoss`.
+    The RNN transducer loss is a prototype feature, see `here <https://pytorch.org/audio>`_ to learn more about the nomenclature. It is only available within the nightlies, and also needs to be imported explicitly using: :code:`from torchaudio.prototype.transducer import TransducerLoss`.
```

Comment:
let's keep the original names `rnnt_loss` and `RNNTLoss`, so i don't expect the doc to need to change

Reply:
I am okay with those names, but please do not reason on the ease of doc update. Giving a name with proper level of descriptiveness is far more important than reducing the doc change.
If this turns out to be a bad name, we should change them and we should do so before moving them out of prototype.